ansible_builder/constants.py

```diff
@@ -12,6 +11,11 @@
 base_roles_path = '/usr/share/ansible/roles'
 base_collections_path = '/usr/share/ansible/collections'
 
+build_arg_defaults = dict(
+    ANSIBLE_RUNNER_BASE_IMAGE='quay.io/ansible/ansible-runner:devel',
```

Comment:
what are your thoughts on dropping 'base' here and using

   ANSIBLE_RUNNER_IMAGE
   
This is in-line with PYTHON_BUILDER_IMAGE below. because there is a PYTHON_BASE_IMAGE today and we can avoid overloading the idea of a 'base' image.

Reply:
This also comes into play with ansible-base / ansible-core too.  So much base!